library/src/main/java/com/arlib/floatingsearchview/FloatingSearchView.java

```diff
@@ -1814,7 +1814,8 @@ public Parcelable onSaveInstanceState() {
 
         savedState.suggestions = mSuggestionsAdapter.getDataSet();
         if (mSuggestionsAdapter.isReversed()) {
-            Collections.reverse(savedState.suggestions);
```

Comment:
This introduced a bug that caused that when device moves the Activity to background, the suggestions list is reversed. Then, the suggestion adapter does not repaint the suggestions, so when the activity return to foreground, if the user selects the first element, the adapter returns that the last one was clicked.